app/forms/state_file/taxes_owed_form.rb

```diff
@@ -88,8 +89,11 @@ def date_electronic_withdrawal_within_range
       end
     end
 
+    def owed_amount
+      intake&.calculated_refund_or_owed_amount&.abs
```

Comment:
[dust|sand] - I know this was the previous syntax as well so I'm presuming all is well in practice. But it still feels off to use `abs` when only owing taxes is valid for this screen. Would multiplying by `-1` be more precise? `abs` could coalesce a refund into a positive value too. I'm guessing having a refund by the point we get to this line is impossible because of upstream logic, but if that's true then `* -1` is what we want anyway, right?